packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -46,6 +46,12 @@ export class VaTelephone {
    */
   @Prop() tty?: boolean = false;
 
+  /**
+   * Indicates if this is a number meant to be used 
+   * to text.
+   */
+  @Prop() sms?: boolean = false;
```

Comment:
Is there a reason why we would go with a prop for this instead of going into "SMS mode" if the contact length is 5 or 6 digits?